provider/cluster_rosa_classic_resource.go

```diff
@@ -1033,6 +1054,18 @@ func (r *ClusterRosaClassicResource) Update(ctx context.Context, request tfsdk.U
 		return
 	}
 
+	err = r.updateIngress(ctx, state, plan)
```

Comment:
Prefet to call it before calling populate for 2 reasons: 
1. I'm working on migration to TF SDK V2 and there we will not have a struct (we will have a similar function that will get the connection )
2. I don't think this is the mission of apopulate_state...

Reply:
Actually populate here populates state before update and according to it decides if we need to update or not. I don't think i can run this before first populate